cmd/snapd-aa-prompt-listener/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
Nitpick: isn't this file missing a license header?
Nitpick^3: Maybe a small description about the command itself?